### PR TITLE
Additional welsh translations

### PR DESCRIPTION
**Pull request by hcarslaw**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the keep-helm label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/tasklistresponse/allegations-of-harm-and-violence/check-your-answers/helpers/satetyConcernHelper.ts [hunk 1]
```diff
@@ -28,7 +30,10 @@ export const HTMLParser = (keys, FoundElement: ANYTYPE, bodyHtml, userCase, type
         Array.isArray(FoundElement['childrenConcernedAbout']) &&
         FoundElement['childrenConcernedAbout'][0] === 'All the children in application'
       ) {
-        bodyHtml += HTML.LIST_ITEM + FoundElement['childrenConcernedAbout'][0] + HTML.LIST_ITEM_END;
+        bodyHtml +=
+          HTML.LIST_ITEM +
+          (language === 'cy' ? 'Pob plentyn yn y cais' : FoundElement['childrenConcernedAbout'][0]) +
```
Comment (vivek-sekhar) [inline src/main/steps/tasklistresponse/allegations-of-harm-and-violence/check-your-answers/helpers/satetyConcernHelper.ts:35]: Can u not hard code it. Pls use infrastructure to handle translations

File: src/main/steps/tasklistresponse/allegations-of-harm-and-violence/check-your-answers/mainUtil.ts [hunk 2]
```diff
@@ -107,33 +114,38 @@ export const SafetyConcerns_child = (
...
     ?  HTML.RULER +  HTML.H4 +  keys['details'] + HTML.H4_CLOSE + userCase['PRL_c1A_policeOrInvestigatorOtherDetails']
     :  '' ;
     /**
    * @PRL_c1A_childAbductedBefore session Values
    */
     let PRL_c1A_childAbductedBefore = '';
-    PRL_c1A_childAbductedBefore += userCase?.['PRL_c1A_passportOffice'];
+    PRL_c1A_childAbductedBefore += (userCase?.['PRL_c1A_passportOffice'] === YesOrNo.YES ? getYesNoTranslation(language, YesOrNo.YES, 'oesTranslation') : getYesNoTranslation(language, YesOrNo.NO, 'oesTranslation'));
+    const relativesTranslation = (relative) => {
+      relative = relative === 'mother' && language === 'cy' ? 'mam' : relative;
```
Comment (vivek-sekhar) [inline src/main/steps/tasklistresponse/allegations-of-harm-and-violence/check-your-answers/mainUtil.ts:127]: pls use translation infrastructure and not hard code.

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/tasklistresponse/allegations-of-harm-and-violence/check-your-answers/mainUtil.ts [hunk 3]
```diff
@@ -60,14 +62,22 @@ export const SafetyConcerns = (
...
+  userCase: Partial<CaseWithId>,
+  language: string
 ): SummaryList | undefined => {
   const childSafetyConcerns = userCase.hasOwnProperty('PRL_c1A_concernAboutChild')
     ? userCase['PRL_c1A_concernAboutChild']?.map(
         concern => HTML.NESTED_LIST_ITEM + keys[concern] + HTML.NESTED_LIST_ITEM_END
       )
     : '';
   let subFields = userCase['PRL_c1A_concernAboutChild'] as ANYTYPE;
+  const childConcernKey = (field) => language === 'cy' ? keys['againstChild']
```
[the remark this answers is not in the document]
Comment (hcarslaw) [reply src/main/steps/tasklistresponse/allegations-of-harm-and-violence/check-your-answers/mainUtil.ts]: The structure of the welsh translation for againstChild is different than the english so I used againstChild for the whole welsh translation but not for english. Have changed to have two separate keys for child and respondent concerns

File: src/main/steps/tasklistresponse/allegations-of-harm-and-violence/check-your-answers/helpers/satetyConcernHelper.ts (hunk 1 above)
Comment (hcarslaw) [reply src/main/steps/tasklistresponse/allegations-of-harm-and-violence/check-your-answers/helpers/satetyConcernHelper.ts]: Changed to use translation infrastructure

**Pull request merged**